jose-util/README.md

```diff
@@ -0,0 +1,54 @@
+# JOSE CLI
+
+The `jose-util` command line utility allows for encryption, decryption, 
+signing and verification of JWE/JWS message on the command line. It's main
+purpose is to facilitate dealing with JWE/JWS messages when testing or
+debugging.
+
+## Usage
+
+The utility has various subdommands. Valid commands are `encrypt`, `decrypt`,
+`sign`, `verify` and `expand`. Examples for each command can be found below.
+
+Input and output files can be specified via the `--in` and `--out` flags.
+Either flag can be omitted, in which case `jose-util` uses stdin/stdout for
+input/output respectively. By default each command will output a compact
+message, but it's possible to get the full serialization by supplying the
+`--full` flag.
+
+Keys are specified via the `--key` flag. Supported key types are naked RSA/EC
+keys and X.509 certificates with embedded RSA/EC keys. Keys must be in PEM
+or DER formats.
+
```

Comment:
Can you include a sentence or two explaining that there are also `--alg` and `--enc` flags that influence the JWE "alg" and "enc" parameters?